src/main/java/br/com/caelum/viagens/aeronaves/model/Assento.java

```diff
@@ -0,0 +1,61 @@
+package br.com.caelum.viagens.aeronaves.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+
+import org.springframework.util.Assert;
+
+@Entity
+public class Assento {
```

Comment:
Aqui vai ser legal na hora da compra... Pq o assento ele pode estar comprado ou não :). To so imaginando o futuro... E aí a gente vai ter que decidir se aceita overbooking e tal... que nem as companhias de hoje em dia hehe